app/layout.tsx

```diff
@@ -29,7 +29,7 @@ export default function RootLayout({
         <body className="min-h-dvh h-full items-center flex flex-col bg-background text-text w-screen">
           <Providers>
             <Header />
-            <main className="w-full m-x-auto items-center flex flex-col pb-8 h-full min-h-[calc(100dvh-var(--header-height)-var(--page-padding-bottom))]">
+            <main className="w-full overflow-x-clip m-x-auto items-center flex flex-col pb-8 h-full min-h-[calc(100dvh-var(--header-height)-var(--page-padding-bottom))]">
```

Comment:
Is this change in scope?

Reply:
actually has nothing to do with this task but it does fix an issue I noticed

Reply:
yeah the lockfile version's been bumped automatically so the diff is kinda huge.